geode-redis/src/integrationTest/java/org/apache/geode/redis/general/ExpireIntegrationTest.java

```diff
@@ -366,4 +371,12 @@ public void CallingExpireOnAKeyThatAlreadyHasAnExiprationTime_ShouldUpdateTheExp
     Long timeToLive = jedis.ttl(key);
     assertThat(timeToLive).isGreaterThan(21);
   }
+
+  @Test
+  public void should_passivelyExpireKeys() {
+    jedis.sadd("key", "value");
+    jedis.pexpire("key", 100);
+
+    GeodeAwaitility.await().until(() -> jedis.keys("key").isEmpty());
```

Comment:
To check my understanding - the jedis.keys() command will just get the keyset, but since it doesn't get the values of the keys it doesn't trigger the active expiration. Right?

Reply:
right